src/common/proc.c

```diff
@@ -94,6 +94,7 @@ crt_proc_prop_entries(crt_proc_t proc, crt_proc_op_t proc_op, daos_prop_t *prop)
 		    entry->dpe_type == DAOS_PROP_CO_OWNER ||
 		    entry->dpe_type == DAOS_PROP_PO_OWNER_GROUP ||
 		    entry->dpe_type == DAOS_PROP_CO_OWNER_GROUP ||
+		    entry->dpe_type == DAOS_PROP_PO_PERF_DOMAIN ||
```

Comment:
I am thinking why define PERF_DOMAIN prop type as string? 
How about take the same method as DAOS_PROP_CO_REDUN_LVL prop that defined as integer type which can directly map to "enum pool_comp_type", and the default value is PO_COMP_TP_ROOT (255), and can with similar valid value check as "C.DAOS_PROP_ENTRY_REDUN_LVL" in propHdlrMap().
By that method can make the mapping be consistent as DAOS_PROP_CO_REDUN_LVL as both them are just a domain level.
How do you think?

Reply:
Or allow user to input a string (valid range in "struct pool_comp_type_dict comp_type_dict::td_name"), but in C backend code probably better be same as DAOS_PROP_CO_REDUN_LVL prop that use an integer type so can directly use. 
ask user to use integer seems also looks fine and is a little simpler (just same as "--properties=rd_fac:1,rd_lvl:2" as explained at above)

Reply:
I think make it integer makes sense, thanks!